react/features/chrome-extension-banner/components/ChromeExtensionBanner.web.js

```diff
@@ -294,7 +304,7 @@ const _mapStateToProps = state => {
 
     return {
         chromeExtensionUrl: bannerCfg.url,
-        chromeExtensionsInfo: bannerCfg.chromeExtensionsInfo || [],
+        chromeExtensionsInfo: bannerCfg.chromeExtensionsInfo,
```

Comment:
Doesn't this complicate the code more because you need to check if it's an array?

Reply:
Ah, I think I get it, you'll get a new array every time so it will trigger a re-render. In that case, how about having a `const emptyArray = [];` at the top of the file and using that?

Reply:
Actually during the implementation I was thinking of doing the same thing. But even the current implementation practically adds 2 additional checks... Which is not so complicated I'd say?

Defining the empty array looked a little bit weird to me...

I'd rather leave it as it is if you don't insist on using the empty array const.

Reply:
Sure, leave it.  For the record: i think the config should take care of filling in default values.